spec/models/thermometer_widget_spec.rb

```diff
@@ -1,41 +1,25 @@
 describe ThermometerWidget do
 
-  let(:content) { {
+  let(:params) { {
     goal: 10000,
     count: 500,
-    autoincrement: true
+    autoincrement: true,
+    page_display_order: 1
   } }
-  let(:string_content) { {
-    goal: "12345",
-    count: "45",
-    autoincrement: "0"
-  } }
```

Comment:
Looking at the answer [here](http://stackoverflow.com/questions/20373954/postgresql-json-querying-in-9-3-on-nested-payloads) is the type of queries I'm imagining, though it looks like you have to specify it's an int at query time regardless to be able to do comparisons.